tools/src/bin/collect.rs

```diff
@@ -498,6 +615,52 @@ async fn run_collection_generic<V: vdaf::Collector, Q: QueryTypeExt>(
 where
     V::AggregateResult: Debug,
 {
+    let collection = new_collector(options, vdaf, http_client)?
+        .start_collection(query, agg_param)
+        .await
+        .map_err(|err| Error::Anyhow(err.into()))?;
+    println!("Job ID: {}", collection.collection_job_id());
```

Comment:
Could we provide machine-parseable output here? Either JSON as discussed elsewhere, or just a flag which causes this command to only output the collection job ID as it would be used as a later input, i.e. without the `Job ID: ` prefix.

I think this is important as these two new commands would be used together, most likely in a scripted way.

If you're planning on doing this as part of the eventual JSON-output-for-everything change, IMO we can wait 'til then.

Reply:
> If you're planning on doing this as part of the eventual JSON-output-for-everything change, IMO we can wait 'til then.

This was my plan, yes.